platforms/documentation/docs/src/docs/userguide/reference/structuring-builds/isolated_projects.adoc

```diff
@@ -108,8 +108,6 @@ These constraints are not final and can change at any time.
 When building from the command line (i.e. for task execution) with Isolated Projects enabled, projects are now configured in parallel.
 
 Parallel configuration does not yet support Configuration on Demand. All projects will be configured, even those which have no tasks to run.
-If this tradeoff is undesirable and you want projects to configure serially with Configure on Demand, set the
-`org.gradle.internal.isolated-projects.configure-on-demand.tasks` system property to `true`.
```

Comment:
💭 Why remove this instead of changing it to the new flag?

Reply:
I removed it mostly because this paragraph is part of an improvised changelog that we currently have on the IP page and because the option is internal. Since it's a changelog, it does not feel right to update the entry (it won't be applicable to 8.11). But I am also reluctant to add a new entry with internal options. As a guideline, we should not include internal options in the public documentation.

The internal options are intended for ourselves and early adopters with whom we work directly. They can break and change at any time, and without any notice. I did leave a message in the code to notify about the name change, for anyone who is already using it.

If we want to make the properties public (even if still `.unsafe.`), that's a separate conversation.